blocks-config/forms/class-uagb-forms.php

```diff
@@ -78,13 +93,23 @@ private function recursive_inner_forms( $blocks_array, $block_id ) {
 					continue;
 				}
 				if ( isset( $blocks['blockName'] ) && ( 'uagb/forms' === $blocks['blockName'] || 'core/block' === $blocks['blockName'] ) ) {
-					if ( ! empty( $blocks['attrs'] ) && isset( $blocks['attrs']['block_id'] ) ) {
+					if ( isset( $blocks['attrs']['ref'] ) ) {
+						$id                      = $blocks['attrs']['ref'];
+						$reusable_blocks_content = $this->reusable_block_content_on_page( $id );
+						$blocks['attrs']         = $this->recursive_inner_forms( $reusable_blocks_content, $block_id );
+					}
+					if ( ! empty( $blocks['attrs'] ) && isset( $blocks['attrs']['block_id'] ) && $blocks['attrs']['block_id'] === $block_id ) {
```

Comment:
```suggestion
					if ( ! empty( $blocks['attrs']['block_id'] ) && $blocks['attrs']['block_id'] === $block_id ) {
```

should we try this assuming $block_id  is not acceptable as 0

Reply:
Resolved